ui/app/styles/components/popup-menu.scss

```diff
@@ -62,7 +62,7 @@
 
```

Comment:
Example of what this effects is the blue "Client count" in the screenshot. 
![image](https://user-images.githubusercontent.com/6618863/231283418-e6ff1e88-e3d7-4876-98d0-c5dbf1d544c9.png)